Dev/full width template

luisdelacruzdev opened the pull request.

luisdelacruzdev committed:
style the start post type template

luisdelacruzdev committed:
Trigger multi-dev env creation

luisdelacruzdev committed:
load portions of html on start template conditionally.
Allow shortcode use on the modal for start template

luisdelacruzdev committed:
add fallback when image is not uploaded to start template

luisdelacruzdev committed:
adjust start template width

luisdelacruzdev committed:
move inline js file to main js

luisdelacruzdev committed:
style full page template

cjcodes commented:
Multidev  created successfully! [http://pr-22-onelove.pantheonsite.io/](http://pr-22-onelove.pantheonsite.io/)

luisdelacruzdev committed:
add fallback

luisdelacruzdev committed:
make all custom metaboxes move to top of the page;
change full page template tag to h3;
change the subtitle char limit to 150;
make the subtitle metabox only show on full page template click
